src/styles/blocks/card.scss

```diff
@@ -0,0 +1,109 @@
+.card {
+  margin: 0;
+  font-family: Roboto, serif;
+  text-align: left;
+
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  box-sizing: border-box;
+  padding: 32px 16px 16px;
+  width: 200px;
+
+  border: 1px #f3f3f3 solid;
+
+  &__image {
+    height: 134px;
+    width: 160px;
+    padding-bottom: 40px;
+    align-self: center;
+  }
+
+  &__title {
+    margin: 0;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    padding-bottom: 4px;
+    color: $main-color;
+  }
+
+  &__code {
+    margin: 0;
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+    padding-bottom: 16px;
+    color: $secondary-color;
+  }
+
+  &__review {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    align-items: center;
+    padding-bottom: 24px;
+
+    &-amount {
+      margin: 0;
+      font-size: 10px;
+      font-weight: 400;
+      line-height: 14px;
+      text-align: right;
+      color: $main-color;
+      display: flex;
+    }
+  }
+
+  &__price {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    align-items: center;
+    padding-bottom: 16px;
+
+    &-label {
+      margin: 0;
+      font-size: 12px;
+      font-weight: 400;
+      line-height: 18px;
+      text-align: left;
+      color: $secondary-color;
+    }
+
+    &-amount {
+      margin: 0;
+      font-size: 16px;
+      font-weight: 700;
+      line-height: 18px;
+      text-align: right;
+      color: $main-color;
+    }
+  }
+
+  &__button {
+    box-sizing: border-box;
+    border-radius: 5px;
+    background-color: $blue-color;
+    height: 40px;
+    border: none;
+    color: #fff;
+
+    font-size: 14px;
+    font-weight: 700;
+    line-height: 16px;
+    text-align: center;
+    text-decoration: none;
+    text-transform: uppercase;
+
+    display: flex;
+    align-items: center;
+    justify-content: center;
+
+    &:hover {
+      background-color: #fff;
+      color: $blue-color;
```

Comment:
The variable `$blue-color` is used here again. Ensure that it is defined in your SCSS files. If it's not defined, this will cause a compilation error.